main/utils/nameList.c

```diff
@@ -146,6 +149,16 @@ void initUsernameSystem()
     listLen[1] = ARRAY_SIZE(adjList2);
     listLen[2] = ARRAY_SIZE(nounList);
     _getMacAddress();
+
+    // Initialize
+    swadgeUsername.user = true;
+    size_t blobLen      = 3 * sizeof(int8_t);
```

Comment:
For one, this'd be safer as `sizeof(swadgeUsername.idxs)`, since that's what you're actually writing into. But also if you're only saving three bytes, you may as well use the 32 bit NVS functions. [There's some extra overhead for variable length NVS entries](https://docs.espressif.com/projects/esp-idf/en/v5.2.5/esp32s2/api-reference/storage/nvs_flash.html#structure-of-entry).